dali/python/nvidia/dali/_utils/external_source_impl.py

```diff
@@ -76,7 +76,7 @@ def assert_cpu_sample_data_type(sample, error_str="Unsupported callback return t
     if isinstance(sample, np.ndarray):
         return True
     if types._is_mxnet_array(sample):
-        if sample.ctx.device_type != 'cpu':
+        if sample.context.device_type != 'cpu':
```

Comment:
revert?

Reply:
context is the old name, ctx is a new alias. The code didn't work as-is with MXNet 1.5  (which I still happened to have on my machine).